airflow-core/tests/unit/utils/test_db.py

```diff
@@ -135,7 +140,10 @@ def test_check_migrations(self):
     def test_upgradedb(self, mock_alembic_command):
         upgradedb()
         mock_alembic_command.upgrade.assert_called_with(mock.ANY, revision="heads")
-        assert mock_alembic_command.upgrade.call_count == 2
+        if "FabAuthManager" in conf.get("core", "auth_manager"):
+            assert mock_alembic_command.upgrade.call_count == 2
+        else:
+            assert mock_alembic_command.upgrade.call_count == 1
```

Comment:
We do not know if `FabAuthManager` is the auth manager in this test? It seems to test that in a test, we should know. Why do not we parametrize this test, and run it with simple auth manager and fab auth manager, so that we test both conditions